apps/tasks/src/jobs/integrations/media-server.ts

```diff
@@ -0,0 +1,46 @@
+import { decryptSecret } from "@homarr/common";
+import { db, eq } from "@homarr/db";
+import { items } from "@homarr/db/schema/sqlite";
+import { JellyfinIntegration } from "@homarr/integrations";
+import { createCacheChannel } from "@homarr/redis";
+
+import { EVERY_5_SECONDS } from "~/lib/cron-job/constants";
+import { createCronJob } from "~/lib/cron-job/creator";
+
+export const mediaServerJob = createCronJob(EVERY_5_SECONDS).withCallback(async () => {
```

Comment:
Are you sure that you want to run this every 5 seconds. Seems quite expensive

Reply:
Isn't Redis all memory? Writing every 5 seconds to memory doesn't seem excessive to me.

Reply:
Okay sure, just wanted to check because we add the 5 seconds thing to test it normally. But I guess it will be fine. Otherwise we can adjust it later easily